<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElectronicalDiary</class>
 <widget class="QDialog" name="ElectronicalDiary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainResources.qrc">
    <normaloff>:/Icons/MainResources/PGEE_NEW_LOGO.ico</normaloff>:/Icons/MainResources/PGEE_NEW_LOGO.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Предмет:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="SubjectBox"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="5">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="styleSheet">
        <string notr="true">th, td background-color: rgb(255, 255, 0);
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RefreshButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="MainResources.qrc">
         <normaloff>:/Icons/MainResources/refresh.ico</normaloff>:/Icons/MainResources/refresh.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Клас:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ClassBox">
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Въведете име на ученик:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NameLine"/>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Име на ученик:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="StudentNameBox"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Клас със букви:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ClassLBox">
       <item>
        <property name="text">
         <string>А</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Б</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>В</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Г</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Д</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="NameFiltrationToggle">
     <property name="text">
      <string>Търсене по име</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="FiltrationToggle">
     <property name="text">
      <string>Без филтрация</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="MainResources.qrc"/>
 </resources>
 <connections/>
</ui>
